cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/ProvisionerSelection/ProvisionerSelection.scss

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../../../../styles/variables.scss";
+
+$box-width: 410px;
+$box-height: 300px;
+$icon-size: 93px;
+
+.profile-create-provisioner-selection {
+  .provisioner-selection-container {
+    margin: 30px 50px;
+
+    .selection-container-label {
+      margin-bottom: 0;
+    }
+
+    .provisioner-selections {
+      display: flex;
```

Comment:
Do you want to try this,

```
.provisioner-selection {
   display: flex;
    display: grid;
    grid-template-columns: repeat(auto-fit, minmax(410px, auto));
    grid-auto-rows: 300px;
    grid-gap: 30px;
    padding: 30px 0;
    flex-wrap: wrap;
}
```

the fallback `flex` display should not make it awkward.

For this style to work correctly the provisioner-box also needs to be modified a little bit,

```
.provisioner-box {
    min-width: 410px;
    min-height: 300px;
    border: 1px solid #999999;
    color: #333333;
    margin: 30px;
    display: flex;
    align-items: center;
    justify-content: center;
}
```

This will make the layout like this,

![screen shot 2018-04-26 at 5 38 18 pm](https://user-images.githubusercontent.com/1452845/39338853-5783aeb4-497a-11e8-9c69-6fd34f8c14ab.png)
 or
![screen shot 2018-04-26 at 5 42 40 pm](https://user-images.githubusercontent.com/1452845/39338855-5c2693be-497a-11e8-863b-b8951535b0b4.png)

instead of centered like this,
![screen shot 2018-04-26 at 5 38 27 pm](https://user-images.githubusercontent.com/1452845/39338863-633a104a-497a-11e8-8289-de0e6eae6785.png)

Reply:
Talked with @ajainarayanan, we decided to remove the usage of `flex` for now, since we want the box to be left aligned when there's only one, but we're also not using `grid` yet since it was not rendering correctly on Safari <11